frontend/src/components/GalleryComponent/Gallery.jsx

```diff
@@ -25,10 +26,11 @@ const Gallery = () => {
       return <li key={image.id}>{<img src={image.url} alt={image} />}</li>;
     });
     return (
-      <section
-        className="m-auto sm:container md:w-11/12 container mt-35 text-center"
-        style={{ marginTop: '5.3125rem' }}
-      >
+      // <section
+      //   className="m-auto sm:container md:w-11/12 container mt-35 text-center"
+      //   style={{ marginTop: '5.3125rem' }}
+      // >
+      <Container padding={CONTAINER_OPTIONS.PADDING.BIG} margin={CONTAINER_OPTIONS.MARGIN.TOW_SIDES} classNames="text-center">
```

Comment:
- please change the padding type to be the default instead of the big
- and fix `classnames` to be `classname`